README.md

```diff
@@ -24,6 +24,8 @@ Use the `italiangrid/voms-testsuite` docker image to run the testsuite.
 | `vo2_issuer`   | VOMS subject DN for the second VO  | DC=org/DC=terena/DC=tcs/C=IT/L=Frascati/O=Istituto Nazionale di Fisica Nucleare/OU=CNAF/CN=vgrid02.cnaf.infn.it  |
 | `vo1_legacy_fqan_enabled`     | Encode FQANs released by first VO as per legacy VOMS | True                                                                                             |
 | `vo2_legacy_fqan_enabled`     | Encode FQANs released by second VO as per legacy VOMS | True                                                                                             |
+| `vo1_legacy_voms`     | First VO is served by a legacy VOMS server | True                                                                                             |
```

Comment:
I would invert the logic here and have the variable named `vo1_is_voms_aa`. I would set the default value to `False` until the majority of server types remains VOMS.